nautobot/core/templates/nautobot_config.py.j2

```diff
@@ -117,6 +117,9 @@ CACHEOPS_DEFAULTS = {"timeout": int(os.getenv("NAUTOBOT_CACHEOPS_TIMEOUT", 900))
 # Set to False to disable caching with cacheops. (Default: True)
 CACHEOPS_ENABLED = is_truthy(os.getenv("NAUTOBOT_CACHEOPS_ENABLED", True))
 
+# Global Celery task timeout (in seconds)
+CELERY_TASK_TIME_LIMIT = os.getenv("NAUTOBOT_CELERY_TASK_TIME_LIMIT", 30 * 60)
```

Comment:
Please double check whether this must be an integer or not and if so cast it to an `int`.